src/pkg/cli/compose/loader.go

```diff
@@ -155,7 +155,7 @@ func (c *Loader) NewProjectOptions() (*cli.ProjectOptions, error) {
 						if inEnv {
 							term.Warnf("Environment variable %q is not used; add it to `.env` if needed", key)
 						} else {
-							term.Debugf("Unresolved variable %s", key)
+							term.Debugf("Unresolved variable %q", key)
```

Comment:
tempted to clarify the unresolved variable here is an _environment_ variable.
```suggestion
							term.Debugf("Unresolved environment variable %q", key)
```